mono/mini/exceptions-amd64.c

```diff
@@ -455,6 +466,7 @@ get_throw_trampoline (MonoTrampInfo **info, gboolean rethrow, gboolean corlib, g
 	mono_arch_flush_icache (start, code - start);
 
 	g_assert ((code - start) < kMaxCodeSize);
+	g_assert_checked (mono_arch_unwindinfo_validate_size (unwind_ops, MONO_MAX_TRAMPOLINE_UNWINDINFO_SIZE));
 
```

Comment:
This could be a normal assert, _checked  () asserts are mainly useful in perf sensitive code.